libcloud/compute/drivers/ec2.py

```diff
@@ -4683,6 +4695,14 @@ def _to_volume(self, element, name=None):
         volId = findtext(element=element, xpath='volumeId',
                          namespace=NAMESPACE)
         size = findtext(element=element, xpath='size', namespace=NAMESPACE)
+        raw_state = findtext(element=element, xpath='status',
+                             namespace=NAMESPACE)
+
+        try:
+            state = self.VOLUME_STATE_MAP[raw_state]
```

Comment:
Minor style thing - doing `self.VOLUME_STATE_MAP.get(raw_state, StorageVolumeState.UNKNOWN)` saves a couple of lines of code. On top of that, code also won't break and require an immediate update if a provider adds a new state (unlikely, but still).